src/sentry/api/endpoints/organization_projects.py

```diff
@@ -172,13 +173,7 @@ def serialize_on_result(result):
                 )
                 return serialize(result, request.user, serializer)
 
-            return self.paginate(
-                request=request,
-                queryset=queryset,
-                order_by=order_by,
-                on_results=serialize_on_result,
-                paginator_cls=OffsetPaginator,
-            )
+            return PaginateArgs(request, queryset, order_by, serialize_on_result)
```

Comment:
```py
return PaginateArgs(
    request=request, 
    queryset=queryset, 
    order_by=order_by, 
    on_results=serialize_on_result,
)
```
isn't really less code than:
```py
return self.paginate(
    request=request,
    queryset=queryset,
    order_by=order_by,
    on_results=serialize_on_result,
    paginator_cls=OffsetPaginator,
)
```

Reply:
Yeah it's a first pass. I eventually want it to only have to return `queryset`